SPECS/systemd/systemd.spec

```diff
@@ -890,6 +890,11 @@ ln -s --relative %{buildroot}%{_bindir}/kernel-install %{buildroot}%{_sbindir}/i
 # Split files in build root into rpms
 python3 %{SOURCE2} %buildroot %{!?want_bootloader:--no-bootloader}
 
+%if 0%{?want_bootloader}
+mkdir -p %{buildroot}/boot/efi/EFI/BOOT
+cp %{buildroot}/usr/lib/systemd/boot/efi/systemd-bootx64.efi %{buildroot}/boot/efi/EFI/BOOT/grubx64.efi
```

Comment:
Similar question here about copy vs linking. Linking will minimize package size bloat.

Reply:
Same reasoning as above - ESP (FAT32) does not support symlink